app/controllers/verify/finance_controller.rb

```diff
@@ -49,7 +49,7 @@ def step_params
     end
 
     def confirm_step_needed
-      redirect_to verify_phone_path if idv_session.financials_confirmation == true
+      redirect_to verify_address_path if idv_session.financials_confirmation == true
```

Comment:
I was trying to replace like-for-like in this PR. Could we do a sweep-up PR later to unify our redirect syntax?